lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -16,19 +16,19 @@ const pageFunctions = require('../../lib/page-functions.js');
 const FULL_PAGE_SCREENSHOT_QUALITY = 30;
 
 /**
- * @param {string} str
+ * @template {string} S
+ * @param {S} str
  */
-function snakeCaseToCamelCase(str) {
-  return str.replace(/(-\w)/g, m => m[1].toUpperCase());
+function kebabCaseToCamelCase(str) {
+  return /** @type {KebabToCamelCase<S>} */ (str.replace(/(-\w)/g, m => m[1].toUpperCase()));
 }
 
 /* c8 ignore start */
 
 // eslint-disable-next-line no-inner-declarations
 function getObservedDeviceMetrics() {
-  // Convert the Web API's snake case (landscape-primary) to camel case (landscapePrimary).
```

Comment:
nice catch!

I think this may have caused issues.

- https://bugs.chromium.org/p/chromium/issues/detail?id=1211704
- I thought there were more about bad emulation state after an audit, but can't find. https://bugs.chromium.org/p/chromium/issues/list?q=component%3APlatform%3ELighthouseAudits